### ISSUE TITLE
wonder the explaination of loss function in classification task

### ISSUE THREAD

**Issue by hyxxx-alt** [hancyran/RepSurf#11]
Thanks for your excellent works !

I had a bit of a problem when reproducing the project. Could you explain the loss function (SmoothClsLoss) used in classification task? I didn't find an introduction to it in the paper.

**Comment by hancyran** [comment]
Thanks for your interest in our work.
We apply label smoothing to the loss function. It is quite a common technique of training on classification task. If you are interested in the mechanism of it, I recommend this [paper](https://arxiv.org/pdf/1906.02629.pdf).
